tfhe/src/high_level_api/integers/keys.rs

```diff
@@ -196,22 +196,6 @@ impl IntegerCompactPublicKey {
         Some(Self { key: Some(key) })
     }
 
-    pub(in crate::high_level_api) fn try_encrypt<T>(
```

Comment:
why is this removed ?

Reply:
Because its not needed anymore

Reply:
That thing is now handled line 51 of [tfhe/src/high_level_api/integers/client_key.rs](https://github.com/zama-ai/tfhe-rs/pull/601/files#diff-db1fc01f72b02f7ba047502431176a57f4df20496246ad61e1e35c01fb7a69ab)

Reply:
ok thanks !